breithbarbot/cropper/2.0/config/packages/breithbarbot_cropper.yaml

```diff
@@ -0,0 +1,3 @@
+# Read the documentation: https://github.com/breithbarbot/Cropper/blob/master/Resources/doc/index.md
+breithbarbot_cropper:
+    mappings:
```

Comment:
Instead of an empty configuration, I suggest to use a lighter version of what you proposed [here in the docs](https://github.com/breithbarbot/cropper/blob/master/Resources/doc/installation.md#step-3-configure-the-breithbarbotcropperbundle) and comment it with a bit of details about how it is supposed to work.